src/tiled/mapobjectmodel.cpp

```diff
@@ -418,6 +433,18 @@ void MapObjectModel::tileTypeChanged(Tile *tile)
     }
 }
 
+void MapObjectModel::emitObjectsChanged(const QList<MapObject *> &objects, const QList<Columns> &columns)
```

Comment:
Thanks for realizing this should indeed by `Columns` rather than `MapObject::Property`! However, please rename the enum to `Column`.